packages/shared/src/lib/featureManagement.ts

```diff
@@ -141,9 +141,13 @@ export class Features<T extends FeatureValue = string> {
   static readonly ShowCommentPopover = new Features('show_comment_popover');
 
   static readonly ScrollOnboardingVersion = new Features(
-    'scroll_onboarding_version',
-    null,
-    [ScrollOnboardingVersion.V1, ScrollOnboardingVersion.V2],
+    'anonymous_users_scroll_home_page',
+    ScrollOnboardingVersion.Control,
```

Comment:
Think we can revert it back to null, people will simply not get it.

Reply:
Resolved here https://github.com/dailydotdev/apps/pull/1629#discussion_r1114001422